mla/analysis.py

```diff
@@ -107,7 +107,7 @@ def _preprocess_ts(self, events: np.ndarray, event_model: models.EventModel
     def evaluate_ts(self, events: np.ndarray, event_model: models.EventModel,  # Super class will never be called... pylint: disable=arguments-differ, too-many-arguments
                     ns: float, gamma: float,
                     n_events:Optional[int] = None,# Python 3.9 bug... pylint: disable=unsubscriptable-object
-                    preprocessing: TsPreprocess) -> np.array:
+                    preprocessing: TsPreprocess = None) -> np.array:
```

Comment:
I realized it was a bad idea to have the preprocess be able to return none. It's better to handle the 0 events case before trying to preprocess.